<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alex R</author>
 <comment>
Copyright (C) 2010 Alex R burbulator@gmail.com

This file is part of Qvernote.

Qvernote is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

Qvernote is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with Qvernote.  If not, see http://www.gnu.org/licenses/.
 </comment>
 <class>QNoteSearchDialogClass</class>
 <widget class="QDialog" name="QNoteSearchDialogClass">
  <property name="windowTitle">
   <string>Search for note</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Words</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leSearchString"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pbStartSearch">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="pbSearchTags">
     <property name="text">
      <string>Tags</string>
     </property>
     <property name="icon">
      <iconset resource="Resources/Icons.qrc">
       <normaloff>:/IconAdd/Tag-32.png</normaloff>:/IconAdd/Tag-32.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="pbSearchNotebook">
     <property name="text">
      <string>All Notes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
